README.md

```diff
@@ -0,0 +1,18 @@
+# Drupal Commerce Documentation
+
+## Local setup
+- Install [Python](https://www.python.org/downloads/)
+- If `pip` is not installed by default, install it from [here](https://pip.pypa.io/en/stable/installing/)
+- Execute `pip install sphinx sphinx-autobuild`
+- Fork this [documentation repository](https://github.com/drupalcommerce/commerce-docs)
+- Clone the forked repo
+- `cd <forked-repo>`
+- Execute `pip install -r requirements.txt`, you might need to execute this command with `sudo`
+- Execute `./build.sh`, this will generate documentation HTML inside `_build/html`
+- `cd` inside `_build/html`
+- Now you can view the documentation locally, you have these options:
+  - You have already installed Python, start `SimpleHTTPServer` using `python -m SimpleHTTPServer 8000`, and visit [http://0.0.0.0:8000/](http://0.0.0.0:8000/) from your browser
```

Comment:
We could just tell people to run `php -S localhost:8080 -t _build/html`, let them figure out anything else.